modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -1102,7 +1102,7 @@ struct GAPI_Streaming_Unit: public ::testing::Test {
 // FIXME: (GAPI_Streaming_Types, XChangeOpaque) test is missing here!
 // FIXME: (GAPI_Streaming_Types,  OutputOpaque) test is missing here!
 
-TEST_F(GAPI_Streaming_Unit, TestTwoVideoSourcesFail)
+TEST_F(GAPI_Streaming_Unit, DISABLED_TestTwoVideoSourcesFail)
```

Comment:
Why is it disabled?

Reply:
Previously there was a restriction on only one true video streaming source, now this restriction is gone

Reply:
Actually it's a good idea to add tests with videos of different length, will doo